### PR TITLE
add column for photographer, improve formatting

### PR DESCRIPTION
**Pull request by gregorwolf**

### CODE REVIEW

File: src/demos/zdemo_calendar.prog.abap [hunk 1]
```diff
@@ -156,9 +160,10 @@ START-OF-SELECTION.
   lo_style_month                        = lo_excel->add_new_style( ).
   " lo_style_month->font->underline       = abap_true.
   " lo_style_month->font->underline_mode  = zcl_excel_style_font=>c_underline_single.
```
Comment (sandraros) [inline src/demos/zdemo_calendar.prog.abap]:
```suggestion
```

File: src/demos/zdemo_calendar.prog.abap [hunk 2]
```diff
@@ -156,9 +160,10 @@ START-OF-SELECTION.
...
-  lo_style_month->font->name            = zcl_excel_style_font=>c_name_roman.
-  lo_style_month->font->scheme          = zcl_excel_style_font=>c_scheme_none.
-  lo_style_month->font->family          = zcl_excel_style_font=>c_family_roman.
+  "lo_style_month->font->name            = zcl_excel_style_font=>c_name_roman.
+  "lo_style_month->font->scheme          = zcl_excel_style_font=>c_scheme_none.
+  "lo_style_month->font->family          = zcl_excel_style_font=>c_family_roman.
+  lo_style_month->alignment->horizontal  = zcl_excel_style_alignment=>c_horizontal_right.
```
Comment (sandraros) [inline src/demos/zdemo_calendar.prog.abap]:
```suggestion
  lo_style_month->alignment->horizontal  = zcl_excel_style_alignment=>c_horizontal_right.
```

**Review by sandraros — CHANGES_REQUESTED** [review]

**Comment by larshp** [comment]
@sandraros we can enable rule https://rules.abaplint.org/commented_code/ to get all the commented code removed in one PR, plus it will give errors to the pull request authors when a new PR is opened

**Review by sandraros — APPROVED** [review]
LGTM 👍

**Pull request merged**